### PR TITLE
[Signalr] - update api versions and create Bicep file

### PR DESCRIPTION
**Pull request by mumian**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@juniwang - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.signalrservice/signalr/main.bicep [hunk 1]
```diff
@@ -0,0 +1,121 @@
+@description('The globally unique name of the SignalR resource to create.')
+param name string = uniqueString(resourceGroup().id)
+
+@description('Location for the SignalR resource.')
+param location string = resourceGroup().location
+
+@description('The pricing tier of the SignalR resource.')
+@allowed([
+  'Free_F1'
+  'Standard_S1'
```
Comment (juniwang) [inline quickstarts/microsoft.signalrservice/signalr/main.bicep:10]: Thanks so much!  We released a new SKU `Premium_P1`, could you please include it?

**Review by juniwang — APPROVED** [review]

**Comment by azure-quickstarts** [comment]
@juniwang - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.signalrservice/signalr/main.bicep [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  'Serverless'
+  'Classic'
+])
+param serviceMode string = 'Default'
+
+@allowed([
+  'true'
+  'false'
+])
+param enableConnectivityLogs string = 'true'
```
Comment (bmoore-msft) [inline quickstarts/microsoft.signalrservice/signalr/main.bicep:38]: let use bool as the type for all the true/false params (and remove allowedValues)

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Comment by azure-quickstarts** [comment]
@juniwang - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Pull request merged**